src/pretix/control/forms/item.py

```diff
@@ -201,6 +201,12 @@ def clean_show_during_checkin(self):
 
         return val
 
+    def clean_type(self):
+        val = self.cleaned_data.get('type')
+        if self.instance:
+            self.instance.clean_type_change(self.instance.type, val)
```

Comment:
I re-read the docs, but could not find why there would not be a `self.instance` at this point. So:
```suggestion
        self.instance.clean_type_change(self.instance.type, val)
```

Reply:
Generally, Django ModelForms are not guaranteed to have an instance available during clean step. For example, a CreateForm usually would have self.instance = None at this point.

In this case, the form does currently always have an instance due to
```
    def get_form_kwargs(self):
        kwargs = super().get_form_kwargs()
        kwargs['instance'] = Question(event=self.request.event)
        return kwargs
```
But that's an implementation detail of the view (the view could also create the event relation when calling save()) and I don't think the form should rely on it.